algorithm/algorithm.md

```diff
@@ -0,0 +1,141 @@
+# MPM Algorithm
+
+## Main Algorithm
+
+The Material Point Method (MPM) algorithm comprises of 4 major parts.
+
+> **Note** 
+> * MPM stores information at the particles and not at the nodes as it is in Finite Element Method. This includes the application of traction at the particles, not at the nodes.
+> * The code is implementing Update Stress First (USF). The other option is to do Update Stress Last (USL). It is done by moving this block of code to the bottom after the particle velocities and displacements have been updated.
+> * The code computes particle mass in the initialization. Even though the density is updated later, the mass is conserved.
+
+1. Initialization
+
+    1. Read all input files and store necessary information
+
+    1. Compute mass of each particle
+        $$ M_p = \gamma V_p $$
+
+1. Solution phase for time step $t$ to $t + \Delta t$
+
+    1. Compute nodal mass 
+        $$ m_I^{t+\Delta t} = \Sigma_p N_I(\textbf{x}_p^t) M_p $$
+
+    1. Compute nodal momentum
+        $$ m\textbf{v}_I^{t+\Delta t} = \Sigma_p N_I(\textbf{x}_p^t) M\textbf{v}_p $$
+
+    1. Compute nodal velocities
+        $$ \textbf{v}_I^{t+\Delta t} = \frac{m\textbf{v}_I^{t+\Delta t}}{m_I^{t+\Delta t}} $$
+
+    1. Compute strain
+        $$ (\boldsymbol{\varepsilon}_p^{t+\Delta t}) = \Sigma_I B_I(\textbf{x}_p^t) \textbf{v}_I^{t+\Delta t} $$
+
+    1. Update stress ($\Delta\sigma_p^{t+\Delta t}$ depends on constitutive model)
```

Comment:
`$$`

Reply:
This is in line

Reply:
The syntax is `$$ <math> $$` or ` {% math %}a \ne 0{% endmath %}`. There is no `$` in the Mathjax plugin. https://plugins.gitbook.com/plugin/mathjax